src/internal/Chooser.tsx

```diff
@@ -0,0 +1,57 @@
+import {IDType} from 'phovea_core';
+import React from 'react';
+import {AView, FindViewUtils, ISelection, ViewWrapper} from 'tdp_core';
+import {ITreeElement, TreeRenderer, viewPluginDescToTreeElementHelper} from 'tdp_ui';
+import {useAsync} from '..';
+import {Range} from 'phovea_core';
+
+
+interface IChooserProps {
+    previousWrapper: ViewWrapper;
+    onOpenView: (viewWrapper: ViewWrapper, viewId: string, idtype: IDType, selection: Range, options?) => void;
+
+}
+
+// tslint:disable-next-line: variable-name
+export const Chooser = ({previousWrapper, onOpenView}: IChooserProps) => {
```

Comment:
```suggestion
export function Chooser({previousWrapper, onOpenView}: IChooserProps) {
```

Why not declaring components as regular function? Then you don't need to disable tslint.